src/Utility/Test/TestFieldFunctionsLR.C

```diff
@@ -53,7 +53,7 @@ TEST(TestFieldFunctions, 2D1PLR)
   scal = f2D_scalar(X);
   x = 2.0, y = 0.5;
   vec = f2D_vec(X);
-  EXPECT_NEAR(scal, x, 1e-14);
+  EXPECT_NEAR(scal, x, 1e-13);
```

Comment:
If the changes in this commit is due to the fix in the previous commit, it should be squashed into that one